pavelib/assets.py

```diff
@@ -733,20 +742,16 @@ def watch_assets(options):
     if tasks.environment.dry_run:
         return
 
-    themes = getattr(options, 'themes', None)
-    theme_dirs = getattr(options, 'theme-dirs', [])
-
+    themes = get_parsed_option(options, 'themes')
+    # option with hyphen(-) is converted to underscore(_) by `cmdopts` so `theme_dirs` will be passed
+    # instead `theme-dirs`. Behavior mentioned in `cmdopts()` of python2.7/site-packages/paver/tasks.py.
```

Comment:
I am not in the favour of duplicating the doc string here and above, maybe we add this information in PR instead.